src/main/groovy/de/dkfz/roddy/tools/RoddyIOHelperMethods.groovy

```diff
@@ -147,6 +149,8 @@ class RoddyIOHelperMethods {
                 stackTrace = exception.getStackTrace()
             if (stackTrace != null)
                 return joinArray(stackTrace, System.getProperty("line.separator"))
+            else
+                return ""
```

Comment:
could put the return at the end of the method then u only need it once

Reply:
I find that more explicit. Actually, in Groovy you may not even use a `return`. I alway find that economic only from the writers perspective, but less clear from the code readers perspective. So I prefer returns, if they are so close to each other like here, and if there are only two/three of them.